charts/iter8lib/templates/_k-cronjob.tpl

```diff
@@ -20,7 +23,7 @@ spec:
             - "/bin/sh"
             - "-c"
             - |
-              iter8 k run --namespace {{ .Release.Namespace }} --group {{ .Release.Name }}
+              iter8 k run --namespace {{ .Release.Namespace }} --group {{ .Release.Name }} -l {{ .Values.iter8lib.logLevel }} --reuseResult
```

Comment:
Maybe I should also change the order so that log level is at the end.

Reply:
sounds good